lib/recommended-rules.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   "ember/no-empty-attrs": "off",
   "ember/no-function-prototype-extensions": "error",
   "ember/no-global-jquery": "error",
+  "ember/no-implicit-injections": "error",
```

Comment:
This would be a major version bump I believe, so we should disconnect the rule addition from the recommended addition IMO.